### ISSUE TITLE
Version 1.0: Communication

### ISSUE THREAD

**Issue by Drewerth** [Drewerth/csgo-morbus#7]
- Team Chat
- Global Dead Chat
- Radios, maybe
- Localized Voice and Text Chat

**Comment by Drewerth** [comment]
In Progress

**Comment by SZOKOZ** [comment]
As an API or stock yes?

**Comment by SZOKOZ** [comment]
Oh, just remembered we were doing morbus_comm.smx etc

**Comment by Drewerth** [comment]
Yeah, just need to make some generic forwards and methods in morbus.smx (Morbus_IsAlien, Morbus_IsHuman, etc)
